scheduler/test/cook/test/mesos/dru.clj

```diff
@@ -89,8 +89,64 @@
             ordered-drus [1.0 1.0 1.0 1.5 4.0 5.5]]
         (is (= ordered-drus
                (map (comp :dru second)
-                    (dru/sorted-task-scored-task-pairs (map-vals (partial sort-by identity (util/same-user-task-comparator))
-                                                          (group-by util/task-ent->user task-ents))
-                                                {"ljin" share "wzhao" share "sunil" share}))))))))
+                    (dru/sorted-task-scored-task-pairs
+                      {"ljin" share "wzhao" share "sunil" share}
+                      (map-vals (partial sort-by identity (util/same-user-task-comparator))
+                                (group-by util/task-ent->user task-ents))))))))))
+
+(deftest test-compute-sorted-task-cumulative-gpu-score-pairs
+  (testing "return empty set on input empty set"
+    (is (= []
+           (dru/compute-sorted-task-cumulative-gpu-score-pairs 25.0 '()))))
+
+  (testing "test1"
+    (let [datomic-uri "datomic:mem://test-score-tasks"
+          conn (restore-fresh-database! datomic-uri)
+          job1 (create-dummy-job conn :user "ljin" :gpus 10.0)
+          job2 (create-dummy-job conn :user "ljin" :gpus 5.0)
+          job3 (create-dummy-job conn :user "ljin" :gpus 25.0)
+          job4 (create-dummy-job conn :user "ljin" :gpus 15.0)
+          task1 (create-dummy-instance conn job1 :instance-status :instance.status/running)
+          task2 (create-dummy-instance conn job2 :instance-status :instance.status/running)
+          task3 (create-dummy-instance conn job3 :instance-status :instance.status/running)
+          task4 (create-dummy-instance conn job4 :instance-status :instance.status/running)
```

Comment:
Should we extract a function for 

```clj
(create-dummy-instance conn % :instance-status :instance.status/running)
``` 
since it's used several times?